src/libraries/Common/src/Interop/Unix/Interop.Odbc.cs

```diff
@@ -0,0 +1,38 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Reflection;
+using System.Runtime.InteropServices;
+
+internal static partial class Interop
+{
+    internal static partial class Libraries
+    {
+        internal const string Odbc32 = "libodbc";
+    }
+
+    internal static partial class Odbc
+    {
+        internal static string GetNativeLibraryName()
+        {
+            if (OperatingSystem.IsMacOS() || OperatingSystem.IsIOS() || OperatingSystem.IsTvOS() || OperatingSystem.IsWatchOS())
```

Comment:
nit: can be expression-bodied method with conditional (also below)

Reply:
Yeah, I wrote them as expression-bodies method originally, but I did not like how the long condition looked. I think the current shape is easier to understand.